presto-native-execution/presto_cpp/presto_protocol/special/BaseHiveColumnHandle.hpp.inc

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace facebook::presto::protocol {
+
+struct BaseHiveColumnHandle : public ColumnHandle {
```

Comment:
Would you add comments to explain why all these classes cannot be auto-generated?

Reply:
@mbasmanova I am working with @imjalpreet to try and automate these as well. We will add comments if we are unable to auto-generate.

Reply:
@majetideepak Got it. Thank you for sharing. Great to know that you are trying to avoid manual overrides. Thanks.